4.4.1 not published on NPM

**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
4.4.0

**Please ask your question**
I cannot find 4.4.1 on NPM, why? :)

In the workflow logs, I see that the npm publish workflow failed:
https://github.com/shaka-project/shaka-player/actions/runs/6070157677/job/16465708155

```
Run npm publish
  npm publish
  shell: /usr/bin/bash -e {0}
  env:
    NPM_CONFIG_USERCONFIG: /home/runner/work/_temp/.npmrc
    NODE_AUTH_TOKEN: ***

> shaka-player@4.4.1 prepublishOnly
> python build/checkversion.py && python build/all.py --force

Error:  Player version does not match git version.
git version: v4.4.1
npm version: 4.4.1
player version: v4.4.0-uncompiled
changelog version: 4.4.1
npm ERR! code 1
npm ERR! path /home/runner/work/shaka-player/shaka-player
npm ERR! command failed
npm ERR! command sh -c -- python build/checkversion.py && python build/all.py --force

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-09-04T07_00_27_651Z-debug-0.log
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release-please.yaml
+++ b/.github/workflows/release-please.yaml
@@ -29,8 +29,6 @@ jobs:
       - uses: actions/checkout@v3
         if: steps.release.outputs.release_created == false
         with:
-          # Checkout the fork, where the PR is generated from.
-          repository: shaka-bot/${{ github.event.repository.name }}
           # Use a special shaka-bot access token for releases.
           token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
       - name: Custom update Player version
@@ -63,8 +61,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          # Check out the origin repo, not the fork, and do it at main, not the
-          # PR branch.
+          # Check out the origin repo, and do it at main, not the PR branch.
           ref: main
           # Use a special shaka-bot access token for releases.
           token: ${{ secrets.RELEASE_PLEASE_TOKEN }}

